assets/lang/ja_JP.ini

```diff
@@ -1224,17 +1224,3 @@ Details = Details
 Dismiss = 無視
 Download = ダウンロード
 New version of PPSSPP available = 新しいバージョンのPPSSPPを利用できます
-
-[VR]
-% of native FoV = % of native FoV
-6DoF movement = 6DoF movement
-Distance to 2D menus and scenes = Distance to 2D menus and scenes
-Field of view scale = Field of view scale
-Force 72Hz update = Force 72Hz update
-Heads-up display scale = Heads-up display scale
-Map controller movements to keys = Map controller movements to keys
-Motion needed to generate action = Motion needed to generate action
-Stereoscopic vision (Experimental) = Stereoscopic vision (Experimental)
-Virtual reality = Virtual reality
-VR camera = VR camera
-VR controllers = VR controllers
```

Comment:
Looks like this section got lost? Please add it back, even if you're not gonna translate it right now (it's only visible for VR users)

Reply:
OK
I made from v1.13.2-2456.
And, merge and tranlate it from v1.13.2-2464.

And....How to upload it???

Reply:
I edited the file on here.
Is it OK??